content/en/agent/proxy.md

```diff
@@ -445,6 +427,13 @@ process_config:
     process_dd_url: http://haproxy.example.com:3836
 ```
 
+To send logs through the proxy, setup the following in the `datadog.yaml` file:
```

Comment:
Should we merge with the above section to have a single one for process, apm and logs?

https://a.cl.ly/xQuW1Kwj

Reply:
done!